server/src/main/java/io/druid/guice/AWSModule.java

```diff
@@ -47,8 +51,21 @@ public AWSCredentialsProvider getAWSCredentialsProvider(final AWSCredentialsConf
 
   @Provides
   @LazySingleton
-  public AmazonEC2 getEc2Client(AWSCredentialsProvider credentials)
+  public AmazonEC2Client getEC2Client(
+      AWSCredentialsProvider credentials,
+      EC2RegionSetter ec2RegionSetter,
+      @JsonProperty("envConfig") EC2EnvironmentConfig envConfig)
```

Comment:
I need some guidance getting this environment config into the `AWSModule`. I'm sure it shows -- but I'm a absolute `guice` novice...